Userland/Applications/PixelPaint/Tools/GradientTool.h

```diff
@@ -55,9 +77,11 @@ class GradientTool : public Tool {
     void draw_gradient(GUI::Painter&, bool with_guidelines = false, const Gfx::FloatPoint drawing_offset = { 0.0f, 0.0f }, float scale = 1, Optional<Gfx::IntRect const&> gradient_clip = {});
     void rasterize_gradient();
     void calculate_gradient_lines();
-    void update_gradient_end_and_derive_start(Gfx::IntPoint const);
-    void translate_gradient_start_end(Gfx::IntPoint const delta, bool update_start_counterwise = true);
-    bool has_gradient_start_end() { return m_gradient_center.has_value() && m_gradient_end.has_value() && m_gradient_start.has_value(); }
+    void calculate_left_right_points(float scale_fraction);
```

Comment:
`left_right` is not the best name; maybe `transversal` or `horizontal` (or something like those) would be better here?